MonoZelda/Controllers/KeyboardController.cs

```diff
@@ -41,38 +51,66 @@ public KeyboardController(CommandManager commandManager)
         };
     }
 
-    // Properties
     public KeyboardState CurrentKeyboardState { get; private set; }
     public KeyboardState PreviousKeyboardState { get; private set; }
 
     public void Update(GameTime gameTime)
     {
         CurrentKeyboardState = Keyboard.GetState();
+        UpdateDirectionalKeysList();
+
+        if (_pressedDirectionalKeys.Any())
+        {
+            Keys mostRecentKey = _pressedDirectionalKeys.Last();
+            _commandManager.Execute(CommandType.PlayerMoveCommand, mostRecentKey);
+        }
+        else
+        {
+            _commandManager.Execute(CommandType.PlayerStandingCommand, Keys.None);
+        }
 
-        // Iterate keyCommandDictionary to check input
         foreach (var keyCommandPair in _keyCommandDictionary)
         {
             (Keys key, bool oneShot) = keyCommandPair.Key;
-            if (!oneShot && (CurrentKeyboardState.IsKeyDown(key) || key == Keys.None))
+
+            if (_directionalKeys.Contains(key))
+                continue;
+
```

Comment:
We could just remove the directional keys from `_keyCommandDictionary`, no? Their logic is handled elsewhere now, and this if condition could be removed if those items are removed from the collection.